### ISSUE TITLE
Visually emphasize where the text input focus is when "editor.renderFinalNewline" is set to false.

### ISSUE THREAD

**Issue by godaygo** [microsoft/vscode#75366]
Firstly, it's very cool that feature from #1027 was implemented. I also agree with the final choice that `editor.renderFinalNewline` should be set to `true` by default because of POLA. But what seemed to me was lost in the long discussions on this topic and is a reasonable proposal made by @geln-84 is to visually highlight where the current focus is, like this 🠚 `U+1F81A`  (or other alternatives https://github.com/atom/atom/issues/7787#issuecomment-332930211):

![highlight](https://user-images.githubusercontent.com/261509/54415257-6c10ea80-4704-11e9-84f8-dea28ada4bba.png)

On the other hand, with the current default set to `true`, it can be seen that when user switches it to `false` he/she understand what the consequences will be.

I am writing this to make sure that that idea was considered. And because it is just a change in this line:

https://github.com/MrZephyr17/vscode/blob/607528b742a4548a18cc20dd2c0dd6be44854a44/src/vs/editor/browser/viewParts/lineNumbers/lineNumbers.ts#L107.

**Comment by alexandrudima** [comment]
I wouldn't add this unless we reach some kind of community buy-in to this.

My thinking is that people configure `editor.renderFinalNewline` to get rid of the extra rendering for the last line. In other words,

#### - with `"editor.renderFinalNewline": true` (default)
<img src="https://user-images.githubusercontent.com/5047891/62141576-16249400-b2ed-11e9-976a-8b35df5ba4e2.png" width="200" />

#### - today with "editor.renderFinalNewline": false` 
<img src="https://user-images.githubusercontent.com/5047891/62141617-2d638180-b2ed-11e9-911d-7fed8255a36b.png" width="200" />

#### - with your proposed suggestion, but with another character that is more widely available in fonts:

<img src="https://user-images.githubusercontent.com/5047891/62141500-f4c3a800-b2ec-11e9-9be2-5bdd3c98014d.png" width="200" />

---

The whole reason to turn the setting to false is to get rid of the `9`... Now we are just replacing the last line number with another character. I believe the feature request was for us to behave like other editors, and other editors don't render anything... So for now I would leave us with not rendering anything, even if the change is trivial...

**Comment by godaygo** [comment]
In fact, I am completely satisfied with the current state of affairs. Just while reading questions on this topic, I noticed that this idea had support among a number of users but there was no feedback. So I decided to create an issue - maybe it’s worth making it customizable? Since there is no traction here and I am completely satisfied with the status quo, this issue can be closed with won't fix resolution.
